commands/create.go

```diff
@@ -458,3 +480,101 @@ func tlsPath(c CommandLine, flag string, defaultName string) string {
 
 	return filepath.Join(mcndirs.GetMachineCertDir(), defaultName)
 }
+
+func updateUserdataFile(driverOpts *rpcdriver.RPCFlags, userdataFlag, customInstallScript string) error {
```

Comment:
can you add comments on this func and next summarizing what they do

Reply:
Yes, I will add a description.